### PR TITLE
Update the README

### PR DESCRIPTION
**Pull request by thecutefinder**
1. GNUify to maintain consistency 
2. Update troubleshooting

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -86,7 +86,7 @@ cd Grasscutter
 .\gradlew jar # Compile
 ```
 
-##### Linux
+##### GNU/Linux
```
Comment (KingRainbow44) [inline README.md:89]: please remove `GNU/`

Comment (thecutefinder) [reply README.md:89]: Why?

Comment (KingRainbow44) [reply README.md:89]: you can do something like Linux (GNU), but i think having the linux first makes more sense

Comment (thecutefinder) [reply README.md]: Done

**Review by KingRainbow44 — APPROVED** [review]

**Pull request merged**